docs/ambassador-program.md

```diff
@@ -0,0 +1,48 @@
+---
+id: ambassador-program
+title: Hashnode Ambassador
+---
+
+## What is it?
+
+The Hashnode Ambassador Program is a way Hashnode rewards its people for spreading the word about Hashnode.
+
```

Comment:
Minor feedback - This should be "program" not "Program" because the name of this initiative is Hashnode Ambassador, not Hashnode Ambassador Program.